internal/feature_test.go

```diff
@@ -96,38 +100,48 @@ func TestScriptBasedInit(t *testing.T) {
 
 	// 1. Try to init a new ScriptBasedSoftwareUpdatable with error for not connected client
 	_, _, err := mockScriptBasedSoftwareUpdatable(t, &testConfig{
-		clientConnected: false, storageLocation: dir, featureID: defaultFeatureID})
+		clientConnected: false, storageLocation: dir, featureID: getDefaultFlagValue(t, flagFeatureID)})
 	if err == nil {
-		t.Fatal("Ditto Client shall not be connected!")
+		t.Fatal("ditto Client shall not be connected!")
 	}
 
 }
 
-// TestSBSUInit tests ScriptBasedSoftwareUpdatable core functionality: init, install and download.
+// TestScriptBasedCore tests ScriptBasedSoftwareUpdatable core functionality: init, install and download.
```

Comment:
TestScriptBasedCore - actually, it's software update core.

Reply:
The naming should be fixed in another PR.